share/functions/__fish_is_git_repository.fish

```diff
@@ -0,0 +1,3 @@
+function __fish_is_git_repository --discription 'Check if current directory is a git repository'
```

Comment:
Now it works after I did `source /usr/share/fish/functions/__fish_is_git_repository.fish`. Perhaps `/usr/share/fish/functions/`is only sourced at login ?